indigo/distribution.yaml

```diff
@@ -780,6 +780,14 @@ repositories:
       url: https://github.com/kejriwalnishant1990/bowpmap_ros.git
       version: master
     status: maintained
+  bowpmap_ros-release:
```

Comment:
This name should not end with `-release` since it is used to refer to this entry by users. Once that suffix is moved the `packages` entries needs to be removed since the repo name matches the single package name.